Mathlib/Logic/Equiv/Set.lean

```diff
@@ -486,7 +486,6 @@ protected noncomputable def image {α β} (f : α → β) (s : Set α) (H : Inje
 #align equiv.set.image Equiv.Set.image
 #align equiv.set.image_apply Equiv.Set.image_apply
 
-@[simp]
```

Comment:
Was this rejected by `simpNF`? If so, in favor of what lemma?

Reply:
`⟨x, ⟨h, rfl⟩⟩ : f x ∈ f '' s` is a proof.

Reply:
That's not supposed to matter... Either way, I'm happy with the comments for now, this PR is clearly a big improvement in general.